### DISCUSSION TITLE
Error on Instagram

### DISCUSSION THREAD

**Discussion by Kory2468** [aclap-dev/video-downloadhelper#515] [Help and Q&A]
### Your question

Any Idea where the problem is?

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "microsoft",
  "channel": "stable",
  "page_url": "https://www.instagram.com/",
  "error": {
    "details": "SideDownload error: : [mov,mp4,m4a,3gp,3g2,mj2 @ 000002db6514b500] trun track id unknown, no tfhd was found\r\n[mov,mp4,m4a,3gp,3g2,mj2 @ 000002db6514b500] error reading header\r\n[in#0 @ 000002db6514b340] Error opening input: Invalid data found when processing input\r\nError opening input file https://scontent.cdninstagram.com/v/t66.30100-16/318612599_1401119487237855_2200954625060618565_n.mp4?_nc_cat=101&ccb=1-7&_nc_sid=9a5d50&efg=eyJ2ZW5jb2RlX3RhZyI6ImlnLXhwdmRzLmNsaXBzLmMyLUMzLmRhc2hfYmFzZWxpbmVfMTA4MHBfdjEiLCJ2aWRlb19pZCI6bnVsbH0%3D&_nc_ohc=MN3XFGByQsIQ7kNvgGpAjS2&_nc_ht=scontent.cdninstagram.com&oh=00_AYAFICYoGQkOalAEe6-CKMQL0CjVCrWqLinNNvkJ7DaZ1Q&oe=66502BA8&bytestart=15719300&byteend=18024995.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_7777371587312607",
    "error": "coapp_failure",
    "id": "coapp_failure_4efa065b-2938-4a9e-8c16-234fe6d0b2bf"
  }
}

**Comment by Mattmjs** [comment]
What was the problem on this one ? The link opens a video without audio similar to Facebook,YouTube etc...
Try the device emulation mode , using **Ctrl + Shift + M**  and  select a mobile or tablet device and see if it makes a difference.
No press **Ctrl + Shift + I**, then press **Ctrl + Shift + M**, the Edge browser won't open with that keyboard shortcut on it's
own which is dumb of Microsoft.
